core/src/main/scala/org/http4s/otel4s/middleware/TypedAttributes.scala

```diff
@@ -86,6 +87,10 @@ object TypedAttributes {
   def userAgentOriginal(userAgent: `User-Agent`): Attribute[String] =
     UserAgentAttributes.UserAgentOriginal(`User-Agent`.headerInstance.value(userAgent))
 
+  /** @return the `error.type` `Attribute` */
+  def errorType(tpe: String): Attribute[String] =
```

Comment:
The `String` fills a bit loose. Should I provide two alternatives? 
```scala
def errorType(status: Status): Attribute[String] = ...
def errorType(error: Throwable): Attribute[String] = ...
```

Reply:
I think so, yeah

Reply:
I think so, yeah

Reply:
Done